src/app/extensions.cpp

```diff
@@ -422,7 +421,14 @@ void Extension::uninstallFiles(const std::string& path,
 
   for (const auto& dir : installedDirs) {
     TRACE("EXT: Deleting directory '%s'\n", dir.c_str());
-    base::remove_directory(dir);
+    try {
+      base::remove_directory(dir);
+    }
+    catch (const std::exception& ex) {
+      (void)ex;
```

Comment:
This is true. Generally `(void)varname` is used to avoid a "unused variable" warning from the compiler, but the `ex` variable is being used with the LOG() functions.